include/verilated_trace.h

```diff
@@ -88,13 +88,15 @@ class VerilatedTraceCommand {
     // These must all fit in 4 bit at the moment, as the tracing routines
     // pack parameters in the top bits.
     enum {
-        CHG_BIT_0 = 0x0,
-        CHG_BIT_1 = 0x1,
```

Comment:
Again think these were clearer before.  (Although I must admit the code looks prettier aligned.)